aiplatform/src/main/java/aiplatform/PredictTextExtractionSample.java

```diff
@@ -0,0 +1,119 @@
+            + "Q: What does LGM stands for?\n"
+            + "A: Last Glacial Maximum.\n"
+            + "\n"
+            + "Q: What did the analysis from the sediment deposits indicate?\n"
+            + "A: Rainfall in the basin during the LGM was lower than for the present.\n"
+            + "\n"
+            + "Q: What are some of scientists arguments?\n"
+            + "A: The rainforest was reduced to small, isolated refugia separated by open forest"
+            + " and grassland.\n"
+            + "\n"
+            + "Q: There have been major changes in Amazon rainforest vegetation over the last how"
+            + " many years?\n"
+            + "A: 21,000.\n"
+            + "\n"
+            + "Q: What caused changes in the Amazon rainforest vegetation?\n"
+            + "A: The Last Glacial Maximum (LGM) and subsequent deglaciation\n"
+            + "\n"
+            + "Q: What has been analyzed to compare Amazon rainfall in the past and present?\n"
+            + "A: Sediment deposits.\n"
+            + "\n"
+            + "Q: What has the lower rainfall in the Amazon during the LGM been attributed to?\n"
+            + "A:\"}";
+    String parameters =
+        "{\n"
+            + "  \"temperature\": 0,\n"
+            + "  \"maxDecodeSteps\": 32,\n"
+            + "  \"topP\": 0,\n"
+            + "  \"topK\": 1\n"
+            + "}";
+    String project = "YOUR_PROJECT_ID";
+    String publisher = "google";
+    String model = "text-bison@001";
+
+    predictTextExtraction(instance, parameters, project, publisher, model);
+  }
+
+  static void predictTextExtraction(
+      String instance, String parameters, String project, String publisher, String model)
+      throws IOException {
+    PredictionServiceSettings predictionServiceSettings =
+        PredictionServiceSettings.newBuilder()
+            .setEndpoint("us-central1-aiplatform.googleapis.com:443")
+            .build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (PredictionServiceClient predictionServiceClient =
+        PredictionServiceClient.create(predictionServiceSettings)) {
+      String location = "us-central1";
+      final EndpointName endpointName =
+          EndpointName.ofProjectLocationPublisherModelName(project, location, publisher, model);
+
+      Value.Builder instanceValue = Value.newBuilder();
+      JsonFormat.parser().merge(instance, instanceValue);
+      List<Value> instances = new ArrayList<>();
+      instances.add(instanceValue.build());
+
+      Value.Builder parameterValueBuilder = Value.newBuilder();
+      JsonFormat.parser().merge(parameters, parameterValueBuilder);
+      Value parameterValue = parameterValueBuilder.build();
+
+      PredictResponse predictResponse =
+          predictionServiceClient.predict(endpointName, instances, parameterValue);
+      System.out.println("Predict Response");
```

Comment:
Should you be printing something from the predictResponse object?

Reply:
I updated the code to print out the response for now.
Currently the client library is not ready to publish a proto to process the response in a structured way.